### PR TITLE
Wishlist added

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Added the wishlist management page.

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.main-container{
+    margin: 8vh 1rem 2rem 1rem;
```
Comment (NeerajVodala) [inline css/wishlist.css]: 8vh 1rem 2rem would work in similar manner
[When three values are specified, the first value applies to the top, the second to the right and left, the third to the bottom.]

File: css/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+.wl-footer{
+    position: fixed;
+}
+
+/* CARD CSS */
+
+.card{
+    height:fit-content;
+    max-width: 50%;
+    padding: 0.8rem 0.8rem 0 0.8rem;
```
Comment (NeerajVodala) [inline css/wishlist.css]: same as above comment

**Review by NeerajVodala — APPROVED** [review]
Everything looks good to me[added a suggestion].
All the best.

File: css/wishlist.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply css/wishlist.css]: oh yes, will change it.

File: css/wishlist.css (hunk 2 above)
Comment (muskan-gupta-0803) [reply css/wishlist.css]: changed

**Comment by muskan-gupta-0803** [comment]
> Everything looks good to me[added a suggestion]. All the best.

thanks for reviewing

**Review by arulk8 — COMMENTED** [review]
Looks good to go.

**Pull request merged**